packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Carbon from '../../../../carbon-graphs/src/js/carbon';
+import '../../css/Graph.module.scss';
+import '../../css/LineGraph.module.scss';
+import utils from '../../../../carbon-graphs/src/js/helpers/utils';
+
+class LineGraph extends React.Component {
+  render() {
+    debugger;
+    return (
+      <div id="canvasContainer">
+        <div id="tooltip" className="tooltip" style={{ display: 'none' }} />
```

Comment:
You likely want to make this id unique on a per-component basis. You can do that by combining 'tooltip' with the provided graphID, or by generating an ID (we typically use the uuid package to do that).